cmd/containerd/main.go

```diff
@@ -195,18 +194,3 @@ func setLevel(context *cli.Context, config *server.Config) error {
 	}
 	return nil
 }
-
-func dumpStacks() {
```

Comment:
Is this not hooked up on Windows yet? I'll make an issue to hook this up. (fine to move this in this PR, it can be moved back later)